doc/tutorial/debug_faq.txt

```diff
@@ -214,10 +214,12 @@ The ``compute_test_value`` mechanism works as follows:
 * ``ignore``: Silently ignore the computation of intermediate test values, if a
   variable is missing a test value.
 
-.. note::
-  This feature is currently incompatible with ``Scan`` and also with ops
-  which do not implement a ``perform`` method.
+Theano also provides the ``print_test_value`` print flag which can hold a boolean value.The default value is ``false``.If ``True``, Theano will override the ‘__str__’ method of its variables to also print the tag.test_value when this is available.
+It can be used as 
+
+.. code-block:: none
 
+    theano.config.print_test_value = 'True'
```

Comment:
I suggest to completely remove the code snipped and just refer to the config section, as this doesn't add anything to what is said in the config section and is going to clutter the documentation.